File: solucoes/2020/nivel jr/fase 1/py/atleta.py
R = int(input())
F = int(input())
C = int(input())

if(F > (R * 3) or C < 95):
    print("diminuir")
elif(F < (R * 2) and C > 97):
    print("aumentar")
else:
    print("manter")

File: solucoes/2020/nivel jr/fase 1/js/atleta.js
var R, F, C

scanf("%d%d%d", "R", "F", "C")

if(F > (R * 3) || C < 95){
    printf("%s", "diminuir")
}
else if(F < (R * 2) && C > 97){
    printf("%s", "aumentar")
}
else{
    printf("%s", "manter")
}